INSTALLATION.md

```diff
@@ -71,7 +71,7 @@ Follow the instructions from section [Setting up REACT_APP_TALAWA_URL in .env fi
 
 ## Setting up REACT_APP_TALAWA_URL in .env file
 
-Make sure to go through [this](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md#accessing-talawa-api) section on `talawa-api's` installation docs.
+Make sure to go through INSTALLATION.md  `accessing-talawa-api` section of [Talawa-API repo](https://github.com/PalisadoesFoundation/talawa-api).
```

Comment:
"Refer to to the `REACT_APP_TALAWA_URL`  section in the INSTALLATION.md file found in the [Talawa-API repo](https://github.com/PalisadoesFoundation/talawa-api)"